src/index.ts

```diff
@@ -89,7 +94,7 @@ async function restoreMiseCache(): Promise<void> {
   core.info(`mise cache restored from key: ${cacheKey}`)
 }
 
-async function setupMise(version: string | undefined): Promise<void> {
```

Comment:
isn't this usually set to `undefined`?

Reply:
I thought so too originally but the `core.getInput` actually returns an empty string when the value for the given variable is not set. This passes the type checks.